src/app/public/src/styles/_variables.scss

```diff
@@ -1,19 +1,16 @@
 @import '_sky-variables.scss';
 
-$stache-font-family-base: $sky-font-family-base !default;
 $stache-font-family-narrow: $sky-font-family-narrow !default;
 $stache-line-height: $sky-line-height-base !default;
+$stache-line-height-headings: 1.1 !default;
 
-$stache-padding-size: $sky-padding !default;
 $stache-grid-columns: 12 !default;
 $stache-grid-padding: $sky-margin-plus-half !default;
-$stache-nav-item-padding: $sky-nav-item-padding !default;
 
 $stache-padding: $sky-padding !default;
 $stache-padding-double: $sky-padding-double !default;
 
-$stache-border-radius: 5px;
-$stache-border-radius-sm: 3px;
+$stache-border-radius: 3px;
```

Comment:
there isn't a variable. But they do have borders documented in their style guide... which... seems to have been updated in the past couple days...?

https://developer.blackbaud.com/skyux2/design/borders

so never mind lol